.github/pull_request_template.md

```diff
@@ -53,3 +53,7 @@ How to retest this PR or trigger a specific build:
 * for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
 * for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] native</b>
 </details>
+
+### CI Status
+
+ You can check Optaplanner organization repositories CI status from [Chain Status webpage](https://kiegroup.github.io/optaplanner/)
```

Comment:
```suggestion
 You can check OptaPlanner repositories CI status from [Chain Status webpage](https://kiegroup.github.io/optaplanner/).
```

Reply:
@Ginxo Technically, `kogito` is the organization in which OptaPlanner repositories reside. OptaPlanner is not an organization. But I guess the URL will show a page with CI status for all OptaPlanner related repositories (`optaplanner-quickstarts`, `optaweb-*`, etc.), right?

Reply:
I''m wondering whether this should be moved to kogito-pipelines repository and to generate as many tabs as we want there, Like Opta Projects Status, Kogito Status, Drools Status...
![image](https://user-images.githubusercontent.com/25130444/160091257-eb0da099-360c-4e8e-ab78-d9c9c1ec482f.png)

what do you thing @yurloc @ge0ffrey @radtriste @danielezonca

Reply:
It's hard to tell without having a access to a prototype where I could test the UX. But it sounds good to have all projects on a single page if it's easy to quickly find the ones that I'm interested in but also have a chance to easily switch to the other ones for some reason.

Reply:
@yurloc @radtriste @ge0ffrey there you have what I propose
https://ginxo.github.io/kogito-pipelines/job/nightly-jobs
4 tabs. Just by adding different generation steps https://github.com/Ginxo/kogito-pipelines/commit/42f2f6562382feb81734fe4548ccda50727b66b4
As you see the tool is very flexible and can be configured up in the way we like, so it is up to us to decide what we want, how and where

Reply:
OptaPlanner is part of the KIE organization. It is NOT part of Kogito (regardless if that's an organization, a platform, an ecosystem or a workflow project).

Reply:
@Ginxo +1 for those 4 tabs. I 'd remove the "pull request status" suffix for simplicity in the tab names